core/utils.py

```diff
@@ -188,7 +189,8 @@ def create_embed(
         )
 
         # footer
-        base_footer = tr("create_embed", "footer")
+        ctx = None
+        base_footer = _(ctx, "Answer for")
```

Comment:
We have the `TranslationContext` object for that (see https://pumpkinpy.readthedocs.io/en/latest/development/translations.html). However, we've not settled on how we're gonna deal with translations where the guild object is not known, like in this case.

Here I think we can cheat a bit: instead of using text, we can do emojis:
```py
base_footer = "📩 "
```
This will get us around that translation problem.

Reply:
resolved in b4e6d7d0a064707ca1c52d507e9d138a2309a43e